### PR TITLE
(DOCSP-39082): Updates local deploy with unified docker image.

### PR DESCRIPTION
**Pull request by corryroot**
I updated the following page for the unified Docker image updates.

- [DOCSP-39082](https://jira.mongodb.org/browse/DOCSP-39082)
- [STAGING](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=66420c1ccfd4ff22fd18afe5)

- [LATEST BUILD LOG](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=66420a3ecfd4ff22fd1846a4)
### Self-Review Checklist

- [ ] Check that the submodule pulled in the right changes (if applicable).
- [ ] [Define](https://wiki.corp.mongodb.com/display/DE/Taxonomy+tagging+instructions) taxonomy [values](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) at top of page.
- [ ] Add genre facets (tutorial or reference), as in this [example PR](https://github.com/10gen/cloud-docs/pull/5042).
- [ ] Add programmingLanguage (if necessary).
- [ ] Add meta keywords (if necessary).
- [ ] Resolve any new warnings or errors in the build.
- [ ] Proofread for spelling and grammatical errors.
- [ ] Check staging for rendering issues.
- [ ] Confirm links are working.

### CODE REVIEW

File: source/atlas-cli-deploy-docker.txt [hunk 1]
```diff
@@ -37,43 +37,40 @@ deployment with Docker.
 
       To learn more, see the `Docker documentation <https://docs.docker.com/desktop/install/mac-install/>`__.
 
-   .. step:: Pull down the latest ``mongodb/atlas`` Docker image.
-
-      **Example:**
+   .. step:: Pull down the latest ``mongodb/mongodb-atlas-local`` Docker image.
```
Comment (fmenezes) [inline source/atlas-cli-deploy-docker.txt:40]: the text `Use the atlas deployments command to create a local deployment with Docker.` is no longer relevant, we don't need `atlas deployments` anymore

File: source/atlas-cli-deploy-docker.txt [hunk 2]
```diff
@@ -309,7 +273,89 @@ of Docker Compose.
...
+Verify the Image Signature
+--------------------------
+
+You can verify the signature of the ``mongodb/mongodb-atlas-local`` 
+Docker image.
+
+.. procedure::
+   :style: normal
+
+   .. step:: Install cosign.
```
Comment (fmenezes) [inline source/atlas-cli-deploy-docker.txt:313]: we are missing steps here, we need to use the tool

File: source/atlas-cli-deploy-docker.txt [hunk 3]
```diff
@@ -309,7 +273,89 @@ of Docker Compose.
...
+      **Example:**
+
+      .. code-block:: sh
+
+         brew install syft
+
+      To learn more, see the :github:`syft README 
+      </anchore/syft/blob/main/README.md>`.
+
+.. step:: Run syft.
```
Comment (fmenezes) [inline source/atlas-cli-deploy-docker.txt:298]: on staging I only see step 1, is is missing spaces?

File: source/atlas-cli-deploy-docker.txt [hunk 4]
```diff
@@ -309,7 +273,89 @@ of Docker Compose.
...
+             - 27017:27017
+       steps:
+         - name: install mongosh
+           run: |
+            curl --output mongosh.deb https://downloads.mongodb.com/compass/mongodb-mongosh_2.2.1_amd64.deb
+            sudo dpkg -i mongosh.deb
+            mongosh --version
+        - run: mongosh 'mongodb://localhost/?directConnection=true' --eval 'show dbs'
 
 Supported Actions
```
Comment (fmenezes) [inline source/atlas-cli-deploy-docker.txt:360]: I don't think `Supported Actions` applies anymore

File: source/atlas-cli-deploy-docker.txt [hunk 5]
```diff
@@ -37,43 +37,40 @@ deployment with Docker.
...
+            :tabid: with-auth
 
-      .. code-block::
+            .. code-block:: sh
 
-         atlas deployments setup --bindIpAll --username root --password root --type local --force
+               docker run -e MONGODB_INITDB_ROOT_USERNAME=user MONGODB_INITDB_ROOT_PASSWORD=pass -p 27017:27017 mongodb/mongodb-atlas-local
 
-      To learn more about the available options, see 
-      :ref:`atlas-deployments-setup`.
```
Comment (JuliaMongo) [inline source/atlas-cli-deploy-docker.txt:70]: Why would it be better to remove "to learn more"?

File: source/atlas-cli-deploy-docker.txt [hunk 6]
```diff
@@ -309,7 +273,89 @@ of Docker Compose.
 
       .. code-block:: sh
 
-         mongosh mongodb://root:{your-password}@localhost:27017/?directConnection=true
+         mongosh "mongodb://user:pass@localhost:27019/?directConnection=true"
+
+Generate a List of Dependencies
+-------------------------------
+
+You can generate a list of the dependances for the
```
Comment (JuliaMongo) [inline source/atlas-cli-deploy-docker.txt:272]: dependencies

File: source/atlas-cli-deploy-docker.txt [hunk 7]
```diff
@@ -309,7 +273,89 @@ of Docker Compose.
...
+   .. step:: Install syft.
+
+      **Example:**
+
+      .. code-block:: sh
+
+         brew install syft
+
+      To learn more, see the :github:`syft README 
+      </anchore/syft/blob/main/README.md>`.
```
Comment (JuliaMongo) [inline source/atlas-cli-deploy-docker.txt:287]: I would move these two lines to be directly after line 281 about the intro to the procedure.

File: source/atlas-cli-deploy-docker.txt [hunk 8]
```diff
@@ -309,7 +273,89 @@ of Docker Compose.
...
+   .. step:: Install cosign.
+
+      **Example:**
+
+      .. code-block:: sh
+
+         brew install cosign
+
+      To learn more, see the `cosign Installation 
+      <https://docs.sigstore.dev/system_config/installation/>`__
```
Comment (JuliaMongo) [inline source/atlas-cli-deploy-docker.txt]: Same here: it is strange to see this after the fact. Instead, I would do this:
You can verify the signature .... using Cosign. To learn more, see ... link to Cosign.

A suggestion: should we move these long http links to the snooty.toml and use shorter replacements in text?

File: source/atlas-cli-deploy-docker.txt (hunk 5 above)
Comment (corryroot) [reply source/atlas-cli-deploy-docker.txt:70]: @JuliaMongo, no longer applies with the unified image.

File: source/atlas-cli-deploy-docker.txt (hunk 7 above)
Comment (corryroot) [reply source/atlas-cli-deploy-docker.txt:289]: @JuliaMongo, I prefer it here. If they can use, brew, they can ignore.

File: source/atlas-cli-deploy-docker.txt (hunk 1 above)
Comment (fmenezes) [reply source/atlas-cli-deploy-docker.txt:37]: `:latest` is optional for any docker image, but we do offer MongoDB versions like `docker pull mongodb/mongodb-atlas-local:7.0.4` and also `docker pull mongodb/mongodb-atlas-local:7.0` or `docker pull mongodb/mongodb-atlas-local:latest`

**Review by fmenezes — APPROVED** [review]

File: source/atlas-cli-deploy-docker.txt [hunk 9]
```diff
@@ -84,12 +78,23 @@ deployment with Docker.
...
+               mongosh "mongodb://localhost:27017/?directConnection=true"
 
+         .. tab:: With Authentication
+            :tabid: with-auth
+
+            .. code-block:: sh
+
+               mongosh "mongodb://user:pass@localhost:27017/?directConnection=true"
 
 Create a Local Atlas Deployment with Docker Compose
```
Comment (fmenezes) [inline source/atlas-cli-deploy-docker.txt:99]: I think we should remove the `Use the atlas deployments command to create a local` part

File: source/atlas-cli-deploy-docker.txt [hunk 10]
```diff
@@ -309,10 +266,98 @@ of Docker Compose.
...
+         brew install cosign
 
-Supported Actions
------------------
+      To learn more, see the `cosign Installation 
+      <https://docs.sigstore.dev/system_config/installation/>`__.
+
+   .. step:: Run the following commands.
+
+      **Example::**
```
Comment (fmenezes) [inline source/atlas-cli-deploy-docker.txt:319]:
```suggestion
      **Example:**
```

**Comment by JuliaMongo** [comment]
LGTM with two qs:
1. Do we need to use the word Example in simple steps? If we do, this is a new practice to me and I am not sure the team is aware of it either. 
I usually just add commands after steps, or, in some cases, I say: For example, run the command similar to the following". 
(This is when the command is too specific, and the users would need to modify it for their needs in any way).

**Review by JuliaMongo — APPROVED** [review]
LGTM

**Comment by corryroot** [comment]
> LGTM with one q:
> 
> 1. Do we need to use the word Example in simple steps? If we do, this is a new practice to me and I am not sure the team is aware of it either.
>    I usually just add commands after steps, or, in some cases, I say: For example, run the command similar to the following".
>    (This is when the command is too specific, and the users would need to modify it for their needs in any way).

We have use **Example:** when there's multiple ways to accomplish the step, but we want to provide a single example.

**Pull request merged**